### DISCUSSION TITLE
Some ideas for improving anti-cheat detection

### DISCUSSION THREAD

**Discussion by iwanttodo1t** [YimMenu/YimMenu#3212] [Ideas]
update : The idea is unrealistic
Delete

**Comment by Rxann** [comment]
Your first one is already a thing(Attacked player while in godmode). Vehicle God Mode isn't detected as in some cases, the game will put your car in godmode.

**Reply by iwanttodo1t** [reply]
the hidden god mode attacks other players and can't detect cheats

**Reply by Rxann** [reply]
No, If a player attacks another player while in godmode, it will be detected.

**Reply by iwanttodo1t** [reply]
Overt God Mode attacks can be detected, but Covert God Mode attacks cannot be detected. I have tested this recently.

**Reply by Rxann** [reply]
> Overt God Mode attacks can be detected, but Covert God Mode attacks cannot be detected. I have tested this recently.

Yes. We can't detect hidden godmode as it is just setting the player health bits.

**Comment by lonelybud** [comment]
i hate when some players in session are protected using semi godmods and they are like ready for pvp

**Reply by Rxann** [reply]
> i hate when some players in session are protected using semi godmods and they are like ready for pvp

enable passive mode or kick them
